src/main/scala/system/RocketTestSuite.scala

```diff
@@ -150,10 +150,17 @@ object DefaultTestSuites {
   val rv64udNames = rv64ufNames + "structural"
   val rv64ud = new AssemblyTestSuite("rv64ud", rv64udNames)(_)
 
+  val rv32uzfhNames = rv64ufNames
+  val rv32uzfh = new AssemblyTestSuite("rv32uzfh", rv32uzfhNames)(_)
+
+  val rv64uzfhNames = rv64ufNames
+  val rv64uzfh = new AssemblyTestSuite("rv64uzfh", rv64uzfhNames)(_)
+
+
```

Comment:
I agree these tests should be added; It seems that there is no option for enabling zfh in rocket chip, thus we are not able to test these test cases now.

Reply:
I added a FP16 config, which passes these tests.

Reply:
> I added a FP16 config, which passes these tests.

Verified. I will send a PR for https://github.com/OpenRigil/rocket-chip/commit/56c109710869c4441f35da9e50069a7d7dcff958 on mill support once this is merged. Should we also include rv32uzfh in CI by introducing a DefaultRV32FP16Config?

Reply:
I don't think that's necessary, I want to avoid a combinatorial explosion in the number of configs